src/Aspire.ProjectTemplates/templates/aspire-empty/AspireApplication.1.ServiceDefaults/Extensions.cs

```diff
@@ -3,7 +3,7 @@
 using Microsoft.Extensions.DependencyInjection;
```

Comment:
Don't we also need to update the PackageReference versions in the project templates as well?

https://github.com/dotnet/aspire/blob/279abee67b851a0196c5197e3b0094b651376849/src/Aspire.ProjectTemplates/templates/aspire-empty/AspireApplication.1.ServiceDefaults/AspireApplication.1.ServiceDefaults.csproj#L15-L16